systemtest: TestAPIKeyInvalidateID is flaky

As seen in https://apm-ci.elastic.co/blue/organizations/jenkins/apm-server%2Fapm-server-mbp/detail/PR-4061/3/pipeline:

```
[2020-08-11T02:51:39.571Z] === RUN   TestAPIKeyInvalidateID

[2020-08-11T02:51:39.571Z]     TestAPIKeyInvalidateID: apikeycmd_test.go:134: 

[2020-08-11T02:51:39.571Z]         	Error Trace:	apikeycmd_test.go:134

[2020-08-11T02:51:39.571Z]         	Error:      	Not equal: 

[2020-08-11T02:51:39.571Z]         	            	expected: []interface {}{"-VVt23MB9kAaqhEWmlVn"}

[2020-08-11T02:51:39.571Z]         	            	actual  : []interface {}{}

[2020-08-11T02:51:39.571Z]         	            	

[2020-08-11T02:51:39.571Z]         	            	Diff:

[2020-08-11T02:51:39.571Z]         	            	--- Expected

[2020-08-11T02:51:39.571Z]         	            	+++ Actual

[2020-08-11T02:51:39.571Z]         	            	@@ -1,3 +1,2 @@

[2020-08-11T02:51:39.571Z]         	            	-([]interface {}) (len=1) {

[2020-08-11T02:51:39.571Z]         	            	- (string) (len=20) "-VVt23MB9kAaqhEWmlVn"

[2020-08-11T02:51:39.571Z]         	            	+([]interface {}) {

[2020-08-11T02:51:39.571Z]         	            	 }

[2020-08-11T02:51:39.571Z]         	Test:       	TestAPIKeyInvalidateID

[2020-08-11T02:51:39.571Z]     TestAPIKeyInvalidateID: apikeycmd_test.go:136: 

[2020-08-11T02:51:39.571Z]         	Error Trace:	apikeycmd_test.go:151

[2020-08-11T02:51:39.571Z]         	            				apikeycmd_test.go:136

[2020-08-11T02:51:39.571Z]         	Error:      	Should be true

[2020-08-11T02:51:39.571Z]         	Test:       	TestAPIKeyInvalidateID

[2020-08-11T02:51:39.571Z]     TestAPIKeyInvalidateID: apikeycmd_test.go:136: 

[2020-08-11T02:51:39.571Z]         	Error Trace:	apikeycmd_test.go:152

[2020-08-11T02:51:39.571Z]         	            				apikeycmd_test.go:136

[2020-08-11T02:51:39.571Z]         	Error:      	Not equal: 

[2020-08-11T02:51:39.571Z]         	            	expected: 401

[2020-08-11T02:51:39.571Z]         	            	actual  : 200

[2020-08-11T02:51:39.571Z]         	Test:       	TestAPIKeyInvalidateID

[2020-08-11T02:51:39.571Z] --- FAIL: TestAPIKeyInvalidateID (2.02s)
```

--- a/systemtest/apikeycmd_test.go
+++ b/systemtest/apikeycmd_test.go
@@ -126,7 +126,9 @@ func TestAPIKeyInvalidateID(t *testing.T) {
 	es := systemtest.NewElasticsearchClientWithAPIKey(attrs["credentials"].(string))
 	assertAuthenticateSucceeds(t, es)
 
-	cmd = apiKeyCommand("invalidate", "--json", "--id", attrs["id"].(string))
+	// NOTE(axw) it is important to use "--id=<id>" rather than "--id" <id>,
+	// as API keys may begin with a hyphen and be interpreted as flags.
+	cmd = apiKeyCommand("invalidate", "--json", "--id="+attrs["id"].(string))
 	out, err = cmd.CombinedOutput()
 	require.NoError(t, err)
 	result := decodeJSONMap(t, bytes.NewReader(out))

